addons/purchase_price_diff/views/product_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_category_property_form" model="ir.ui.view">
+        <field name="name">product.category.property.form.inherit.stock</field>
+        <field name="model">product.category</field>
+        <field name="inherit_id" ref="account.view_category_property_form"/>
+        <field name="arch" type="xml">
+            <field name="property_account_income_categ_id" position="before">
+                <field name="property_account_creditor_price_difference_categ"
+                    domain="[('deprecated','=',False)]" groups="account.group_account_readonly"
+                    attrs="{'invisible':[('property_valuation', '=', 'manual_periodic')]}"/>
```

Comment:
`property_valuation` is a field in `stock_account` and this module only depends on `purchase`, will that cause an issue?